Src/AmrCore/AMReX_Interp_1D_C.H

```diff
@@ -105,6 +106,35 @@ face_linear_interp_x (int i, int /*j*/, int /*k*/, int n, Array4<T> const& fine,
     }
 }
 
+template <typename T>
+AMREX_GPU_HOST_DEVICE AMREX_FORCE_INLINE
```

Comment:
I feel like we should make this non-template, because we only need the `amrex::Real` version and the function body uses `Real` in many places already.